cypress-shared/support/adyen/graphql.js

```diff
@@ -0,0 +1,85 @@
+import { FAIL_ON_STATUS_CODE } from '../common/constants'
+import { updateRetry } from '../common/support'
+
+const config = Cypress.env()
+
+// Constants
+const { vtex } = config.base
+
+export function updateAdyenConnectorSettings({ useAdyenPlatforms = true }) {
```

Comment:
Do we have any testcase where we disable adyenPlatform and test
@shashidhar85

Reply:
No @Syed-Vtex

Reply:
Then let's remove this
{ useAdyenPlatforms = true }
from function params